mava/types.py

```diff
@@ -103,7 +102,7 @@ def discount_spec(self) -> specs.BoundedArray:
         ...
 
     @property
-    def unwrapped(self) -> Environment:
+    def unwrapped(self) -> Any:
```

Comment:
🔥

Reply:
Ye because the unwrapped isn't necessarily a jumanji env, it could be anything e.g smax

Reply:
Yeeesss that's why I like it we're detaching from relaying on jumanji 💃